react/CompositeRow/styles.styl

```diff
@@ -0,0 +1,13 @@
+@require 'utilities/spaces.styl'
```

Comment:
Doesn't requiring that file add the utility classes to the output? I think i might be better to put the spaces variables in `stylus/settings/` and just load that?

Reply:
Thanks ! Fixed in https://github.com/cozy/cozy-ui/pull/1270/commits/a04cc951160ddce9f0fd5afdba4bbfe0e06a81f3